charts/prometheus-postgres-exporter/templates/servicemonitor.yaml

```
@@ -30,6 +33,10 @@ spec:
 {{- if .Values.serviceMonitor.relabelings }}
     relabelings:
 {{ toYaml .Values.serviceMonitor.relabelings | nindent 4 }}
+{{- end }}
+{{- with .Values.serviceMonitor.tlsConfig }}
+    tlsConfig:
+{{- . | toYaml | nindent 6 }}
```

This way is more common:

```suggestion
{{- toYaml . | nindent 6 }}
```

---

I will change that.